external-crates/move/crates/move-vm-runtime/src/unit_tests/value_tests.rs

```diff
@@ -175,93 +168,6 @@ fn global_value_non_struct() -> PartialVMResult<()> {
     Ok(())
 }
 
-#[test]
```

Comment:
Do we / should we maintain a version of these tests for `abstract_memory_size` now?

Reply:
Good question, but I don't think so, as this was only testing that they lined up on these things.